ext/native/thin3d/VulkanQueueRunner.cpp

```diff
@@ -776,7 +776,10 @@ void VulkanQueueRunner::ApplyRenderPassMerge(std::vector<VKRStep *> &steps) {
 					if (steps[j]->dependencies.contains(touchedFramebuffers)) {
 						goto done_fb;
 					}
-					if (steps[j]->render.framebuffer == fb) {
+					if (steps[j]->render.framebuffer == fb &&
+						steps[j]->render.color != VKRRenderPassAction::CLEAR &&
+						steps[j]->render.depth != VKRRenderPassAction::CLEAR &&
+						steps[j]->render.stencil != VKRRenderPassAction::CLEAR) {
```

Comment:
Hm.  If we were planning on merging anyway, couldn't we just toss the previous render pass in the bin?

That is, assuming our dependency tracking is working, doesn't this indicate that the previous render pass is unused and does not affect anything?

-[Unknown]

Reply:
Yes, but we could only do that if all three are clears, which probably is quite unusual.

In this case, our bad ReformatFramebufferFrom only touches color.

Reply:
Right, my bad.  I've seen games do a clear mid scene that probably could be front-loaded, but the overhead of tracking or fixing sounds troublesome indeed.

-[Unknown]